### DISCUSSION TITLE
Snoop Selection button placement

### DISCUSSION THREAD

**Discussion by Nice3point** [jeremytammik/RevitLookup#151] [Polls]
What do you think of the new location of the "Snoop Selection" button? How do you feel about it and what is your opinion on it, should it be kept or made optional? The idea is that Revit automatically opens "Modify" tab, making the Snoop button available at all times

**Comment by ilia7755404** [comment]
It is one of the best features. It is so convenient when you need to clarify class fields or some hidden parameters. No need to suffer anymore clicking on addin tab and finding button to snoop selection. Lookup target audience is developers not common modelers, so  lookup improvement should stick to developers needs. Even if we count amount of clicks to achieve snooping selected element - new solution in modify tab is absolutely winner!

**Comment by ricaun** [comment]
The "Modify" tab should be optional with a toggle button in the setting to enable/disable. If every plugin decides to add a button in the "Modify " tab, in the end gonna be a mess, if the user wants to disable that great. 

"Modify" tab is the most important tab in Revit, and could be helpful when snooping and other times gonna stay that filling a space in my precious "Modify" tab.

I gonna disable 😣

**Comment by Nice3point** [comment]
Added as an option and disabled by default. Thank to everyone who contributed to the survey
